resources/views/pages/lists.blade.php

```diff
@@ -47,10 +47,15 @@
 	@endrole
 
 	@role(["promoci","kampci","kantoorci","board"])
-	<li role="presentation"><a href="#inschrijvingen" aria-controls="inschrijvingen" role="tab"
+	<li role="presentation"><a href="#deelnemers-inschrijvingen" aria-controls="deelnemers-inschrijvingen" role="tab"
 			data-toggle="tab">Inschrijvingen</a></li>
 	@endrole
 
+	@role(["promoci","kampci","board"])
+	<li role="presentation"><a href="#nieuwe-leidingen" aria-controls="nieuwe-leidingen" role="tab"
```

Comment:
Volgens mij is `leidingen` hier niet het juiste meervoud van `leiding`, we zijn toch geen stalen buizen :P 
Zou er `Nieuwe leiding` van maken.

Reply:
Fixed